server/views/serviceProviderReferrals/actionPlan/actionPlanEditConfirmation.njk

```diff
@@ -6,6 +6,8 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             {{ govukBackLink(backLinkArgs) }}
+            {{ govukNotificationBanner(serviceUserNotificationBannerArgs) }}
```

Comment:
why are we adding this here when it should be in `scheduleAppointmentCheckAnswers.njk`?

Need to check what is looks like on page.